web/server/entrypoint.go

```diff
@@ -195,7 +195,7 @@ func (s *robotServer) createWebOptions(cfg *config.Config) (weboptions.Options,
 	if err != nil {
 		return weboptions.Options{}, err
 	}
-	options.Pprof = s.args.WebProfile
+	options.Pprof = s.args.WebProfile || cfg.EnableWebProfile
```

Comment:
[q] So if we run the RDK with `-webprofile` and put `enable_webprofile: false` in the JSON config, the web profile will still be enabled?

Reply:
Yes that's how it currently behaves. I think that behavior makes sense as in if I add the flag explicitly I expected to just work. Happy to change it if we end up finding this to be confusing.

Reply:
Cool; thanks! I think what you have now makes sense 👍🏻.